src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements AreaCalculator, FigureDrawer {

```

Comment:
This class should be abstract and include common state for each figure - color and behaviour - can obtain the area of any figure and are able to draw it.